src/components/sbb-divider/sbb-divider.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { SbbDivider } from './sbb-divider';
+import { newSpecPage } from '@stencil/core/testing';
+
+describe('sbb-divider', () => {
+  it('renders', async () => {
+    const { root } = await newSpecPage({
+      components: [SbbDivider],
+      html: '<sbb-divider />',
+    });
+
+    expect(root).toEqualHtml(`
```

Comment:
Could you add tests for the different states and whether the correct css class is used?

Reply:
Tests added